src/GafferImage/Catalogue.cpp

```diff
@@ -482,8 +513,11 @@ class Catalogue::InternalImage : public ImageNode
 					// Set up an ImageWriter to do the actual saving.
 					// We do all graph construction here in the main thread
 					// so that the background thread only does execution.
+					m_deleteMeta = new DeleteImageMetadata;
+					m_deleteMeta->inPlug()->setInput( m_imageCopy->outPlug() );
+					m_deleteMeta->namesPlug()->setValue( g_isRenderingMetadataName );
 					m_writer = new ImageWriter;
-					m_writer->inPlug()->setInput( m_imageCopy->outPlug() );
+					m_writer->inPlug()->setInput( m_deleteMeta->outPlug() );
```

Comment:
This would be unnecessary if we used `enabledPlug()` to manage metadata existence, once we've fixed copied images to be `isRendering == false`.

Reply:
Removed [here](https://github.com/GafferHQ/gaffer/pull/4018/commits/4cd8d0234fbd19b6f8c425991dcb65b2c4f9ed02).